pkgs/tools/filesystems/ceph/default.nix

```diff
@@ -239,12 +239,14 @@ in rec {
   };
 
   ceph-client = runCommand "ceph-client-${version}" {
-      meta = getMeta "Tools needed to mount Ceph's RADOS Block Devices";
+      meta = getMeta "Tools needed to mount Ceph's RADOS Block Devices/Cephfs";
     } ''
-      mkdir -p $out/{bin,etc,${sitePackages},share/bash-completion/completions}
+      mkdir -p $out/{bin,sbin,etc,${sitePackages},share/bash-completion/completions}
       cp -r ${ceph}/bin/{ceph,.ceph-wrapped,rados,rbd,rbdmap} $out/bin
       cp -r ${ceph}/bin/ceph-{authtool,conf,dencoder,rbdnamer,syn} $out/bin
       cp -r ${ceph}/bin/rbd-replay* $out/bin
+      cp -r ${ceph}/sbin/mount.ceph $out/sbin
```

Comment:
Mhm, on second thought why does this actually go to `sbin`?

Reply:
I'm not so sure. I looked into my nix store and saw other sbin files in sbin folders, so I used this pattern. Do superuser files also go in $out/bin ?

Reply:
We usually symlink sbin to bin. We do not have the separation otherwise.

Reply:
The files in sbin are stripped of their suid bit which removes the use case for sbin.